Dropp/Dropp-swift/Dropp/UserObject.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UserObject.swift
+//  Dropp
+//
+//  Created by Pongsakorn Cherngchaosil on 4/9/17.
+//  Copyright © 2017 Group B. All rights reserved.
+//
+
+import Foundation
+
+class UserObject : NSObject {
+    var userId: String?
+    var location: String?
+    var timestamp: String?
+    var content: String?
+    var text: String?
+    
+    init(pUserId: String, pLocation: String, pTimestamp: String, pContent: String, pText: String) {
+        self.userId = pUserId
+        self.location = pLocation
+        self.timestamp = pTimestamp
+        self.content = pContent
+        self.text = pText
+    }
+    
+    convenience init(random: Bool = false) {
+        if random {
+            let locations = ["Pyramid", "USU", "Rec Center", "Parking Lot", "CECS Building"]
```

Comment:
These need to be latitude/longitude coordinates, like `let locations = ["69,69", "1,0", "6,9"]`